CTF/src/crypto.h

```diff
@@ -0,0 +1,8 @@
+#ifndef CRYPTO_H
```

Comment:
Since this is generated by `build.rs` can we add it to a `.gitignore`?

Reply:
This isn't generated by build.rs, this is referred by the build.rs.